src/main/java/commands/workout/workoutsession/WorkoutSessionSearch.java

```diff
@@ -2,15 +2,34 @@
 
 import commands.Command;
 import storage.workout.Storage;
+import ui.CommonUi;
 import ui.workout.workoutsession.WorkoutSessionUi;
+import workout.workoutsession.WorkoutSessionParser;
 import workout.workoutsession.exercise.Exercise;
 
 import java.util.ArrayList;
 
 public class WorkoutSessionSearch extends Command {
+    private static CommonUi printer = new CommonUi();
+
     @Override
     public void execute(String[] input, ArrayList<Exercise> exercise,
                         String filePath, Storage storage, boolean[] endWorkoutSession) {
```

Comment:
Should the boolean array name sounds more boolean (i.e. hasEndedWorkoutSessions)?

Reply:
good call